impl/src/main/java/slack/SlackResourceImpl.java

```diff
@@ -18,40 +17,41 @@
 import rx.Observable;
 
 import java.util.List;
-import java.util.concurrent.Callable;
 
 import static rx.Observable.empty;
 import static rx.Observable.error;
-import static rx.Observable.fromCallable;
 
 @Singleton
 public class SlackResourceImpl implements SlackResource {
 
     private static final Logger LOG = LoggerFactory.getLogger(SlackResourceImpl.class);
-    private final Slack slack;
+    public static final String USER_EMAIL_FROM_SLACK_IS_NULL_AND_MIGHT_BE_DUE_TO_MISSING_SCOPE_USERS_READ_EMAIL = "User email from slack is null and might be due to missing scope users:read.email";
```

Comment:
I think we should name the constants to convey the gist of the error rather that the exact words.